src/controllers/controller.h

```diff
@@ -141,8 +141,8 @@ class Controller : public QObject {
 
   public:
     // This must be reimplemented by sub-classes desiring to send raw bytes to a
-    // controller.
-    virtual void sendBytes(const QByteArray& data) = 0;
+    // controller. Return true in case of successful completion, false in case of failure.
```

Comment:
```suggestion
    // controller. Return true in case of successful completion, false in case of partial completion or failure.
```
as seems to be case in the Hss case